web/controllers/template_controller.ex

```diff
@@ -6,14 +6,16 @@ defmodule OrgtoolDb.TemplateController do
   alias OrgtoolDb.TemplateProp
 
   @opts [include: "category,template_props"]
-  @preload [:category, :template_props]
+#    @preload [:category, :template_props]
+  @preload [:category]
 
   if System.get_env("NO_AUTH") != "true" do
     plug Guardian.Plug.EnsureAuthenticated, handler: OrgtoolDb.SessionController, typ: "access"
   end
 
   def index(conn, _params, _current_user, _claums) do
     templates = Repo.all(Template)
+    |> Repo.preload(@preload)
```

Comment:
if we preload here we 1) need to mark it as options and 2 probably should make 2 preloads and 2 options, one for index and one for show.

We want more details / values shown when we do a direct read vs a index.